e2e/helpers/puppeter_helper.js

```diff
@@ -1,3 +1,5 @@
+/* global Helper */
```

Comment:
That should not be needed. Make sure you are aligned with master and you have done `yarn install` to get latest linter.

Reply:
👍